### PR TITLE
Improving Attachments Info

### PR DESCRIPTION
**Pull request by bentleycook**
## Description

Adding a bit more information around working with the attachments API. We've had a number of questions arise including those on this issue: https://github.com/trello/api-docs/issues/111.

Hoping that the Glitch project can be a repository for examples of how best to work with this endpoint as it is quite complicated but very powerful.

### CODE REVIEW

File: app/templates/docs/card.html [hunk 1]
```diff
@@ -140,15 +140,15 @@
...
-                <li><code class="docutils literal"><span class="pre">bytes</span></code></li>
-                <li><code class="docutils literal"><span class="pre">date</span></code></li>
-                <li><code class="docutils literal"><span class="pre">edgeColor</span></code></li>
-                <li><code class="docutils literal"><span class="pre">idMember</span></code></li>
-                <li><code class="docutils literal"><span class="pre">isUpload</span></code></li>
-                <li><code class="docutils literal"><span class="pre">mimeType</span></code></li>
-                <li><code class="docutils literal"><span class="pre">name</span></code></li>
-                <li><code class="docutils literal"><span class="pre">previews</span></code></li>
-                <li><code class="docutils literal"><span class="pre">url</span></code></li>
+                <li><code class="docutils literal"><span class="pre">bytes</span></code> - Size of the attcached file in bytes.</li>
```
Comment (mpcowan) [inline app/templates/docs/card.html]: Typo: `attcached `

File: app/templates/docs/card.html [hunk 2]
```diff
@@ -1798,6 +1798,7 @@
   </h2>
   <ul class="simple">
     <p>The attachments endpoint should be used to add an attachment to a card. Cards are limited to having 100 attachments. You can only add a single attachment to a card at a time.</p>
+    <p>There are two primary ways of adding attachments to cards. The first is by attaching a URL directly to a card. To do so, you include the URL as a query parameter to the POST request. The second way requires that you pass in a file object in the POST request. Trello expects that the file object is encoded as multipart/form-data. For examples demonstating how to add attachments check out this <a href="https://glitch.com/~trello-attachments-api">Glitch project</a>!</p>
```
Comment (mpcowan) [inline app/templates/docs/card.html]: I might trade the `!` at the end for something like 🚀  but that's a personal choice 😛

Comment (mpcowan) [inline app/templates/docs/card.html]: Also do you want the Glitch project to open in the same tab or a new tab?

Comment (bentleycook) [inline app/templates/docs/card.html]: 🚀🚀🚀🚀🚀
Done and done.

**Pull request merged**